Database/Repositories/Implementations/PlayerRatingsRepository.cs

```diff
@@ -24,33 +24,31 @@ public async Task<IEnumerable<PlayerRating>> GetAsync(long osuPlayerId)
         return await _context.PlayerRatings.Where(x => x.PlayerId == id.Value).ToListAsync();
     }
 
-    public async Task<PlayerRating?> GetAsync(int playerId, Ruleset ruleset) =>
-        await _context.PlayerRatings.Where(x => x.PlayerId == playerId && x.Ruleset == ruleset).FirstOrDefaultAsync();
-
-    public async Task<int> BatchInsertAsync(IEnumerable<PlayerRating> playerRatings)
+    public async Task<PlayerRating?> GetAsync(int playerId, Ruleset ruleset)
     {
-        var ls = new List<PlayerRating>();
-        foreach (PlayerRating stat in playerRatings)
+        var results = await _context.PlayerRatings
+            .Include(pr => pr.Adjustments)
+            .Where(pr => pr.PlayerId == playerId && pr.Ruleset == ruleset)
+            .Select(pr => new
+            {
+                PlayerRating =
+                    pr, // TODO: Should probably include all RatingAdjustments regardless of type and have web not display initial
```

Comment:
Similar to the above comment, I don't see why we are ever excluding the initial rating to begin with... Besides that shouldn't be API's responsibility. Whoever is requesting the data can do what they want with it and we should try to include the full dataset where reasonable